cmd/util.go

```diff
@@ -147,6 +147,68 @@ func getConfigFiles(v *viper.Viper) map[string]string {
 	return confmap
 }
 
+// getPodSpecFiles finds which of the set of candidate podspec files exist
+func getPodSpecFiles(v *viper.Viper) map[string]string {
```

Comment:
Even if there's a reason to split out pod and unit files, why not just add a name parameter to the config file function and use it for all three cases? I think that's the only real difference between these three get*Files() functions

Reply:
Yes. I agree with that.

Reply:
OK, great, could you make that change please and then I think we'll be good to go!